FirebaseMessaging/Sources/FIRMMessageCode.h

```diff
@@ -47,7 +47,7 @@ typedef NS_ENUM(NSInteger, FIRMessagingMessageCode) {
   kFIRMessagingMessageCodeTokenDelegateMethodsNotImplemented = 2023,         // I-FCM002023
   kFIRMessagingMessageCodeTopicFormatIsDeprecated = 2024,
   kFIRMessagingMessageCodeDirectChannelConnectionFailed = 2025,
-  kFIRMessagingMessageCodeInvalidClient = 2026, // no longer used
+  kFIRMessagingMessageCodeInvalidClient = 2026,  // no longer used
```

Comment:
nit: this whitespace change looks accidental

Reply:
Done. Not sure if it's clang-format, I revert the change and will run the script again to see.